scripts/o3de/o3de/utils.py

```diff
@@ -135,25 +134,20 @@ def run(self) -> int:
         """
         ret = 1
         try:
-            with Popen(self.args, cwd=self.cwd, env=self.env, stdout=PIPE, stderr=PIPE) as process:
+            with Popen(self.args, cwd=self.cwd, env=self.env, stdout=PIPE, stderr=STDOUT) as process:
```

Comment:
Could you update the PR comment to mention why the change to STDOUT?

Reply:
setting `stderr` to `PIPE` forces us to read the error stream at the end of the process. During the polling process, you cannot poll `stderr` because it was blocking. `STDOUT` will redirect to `stdout` which is directed to `PIPE`, so instead we are reporting the error lines as they occur in sequence, not the end